packages/kbn-test/src/functional_tests/cli/start_servers/args.js

```diff
@@ -27,10 +27,8 @@ const options = {
     desc: 'Pass in a config',
   },
   esFrom: {
-    arg: '<snapshot|source>',
-    choices: ['snapshot', 'source'],
-    desc: 'Build Elasticsearch from source or run from snapshot.',
-    default: 'snapshot',
```

Comment:
Does it still default to snapshot?

Reply:
It should I added it to lines: 81-83
if (!userOptions.esFrom) {
     userOptions.esFrom = 'snapshot';
}

Reply:
Not sure I understand what you mean to keep default property here, please clarify.

Reply:
You removed the default property from this option which removed it from the help info for the command. You want to keep the line `default: 'snapshot',`

Reply:
Ok - got it, made the change.  Thanks.